include/wfc/utility/math.hpp

```diff
@@ -4,10 +4,17 @@
 #include <limits>
 #include <type_traits>
 
+#ifndef __has_builtin
+	#define WFC__has_builtin(x) 0
+#else
+#define WFC__has_builtin(x) __has_builtin(x)
```

Comment:
Undefine macro at the end of file is good practice.

Reply:
Added undef at the end of the file.